include/evmc/loader.h

```diff
@@ -103,6 +103,19 @@ evmc_create_fn evmc_load(const char* filename, enum evmc_loader_error_code* erro
 struct evmc_instance* evmc_load_and_create(const char* filename,
                                            enum evmc_loader_error_code* error_code);
 
+/**
+ * Returns the human-readable message describing the most recent error
+ * that occurred in EVMC loading.
+ *
+ * In case any loading function returned ::EVMC_LOADER_SUCCESS this function always returns NULL.
+ * In case of error code other than success returned, this function MAY return the error message.
+ * This function is not thread-safe and not supported on all operating systems.
```

Comment:
Reading this as a user leaves me confused about whether I can call it on my operating system. Maybe specify that it returns NULL if not supported?

Reply:
I will remove the sentence about the operating system.